call_google_translate.py

```diff
@@ -150,6 +151,7 @@ def balanced_brackets(line):
             try:
               translation = translator.translate(notes, src='en', dest=language)
             except Exception:
+              traceback.print_exc()
```

Comment:
Was this for debugging only or did you intend to leave this in here? Did you find the output helpful? I don't really think it's necessary.

Reply:
I committed it accidentially after debugging the translation script. I don't think it's generally necessary unless the translation script becomes broken (which can be due to outdated version, network problems, etc.)